.github/actions/difftest/action.yml

```diff
@@ -0,0 +1,46 @@
+name: Run difftest
+description: Runs difftest
+inputs:
+  flags:
+    description: difftest flags
+    required: true
+
+  target:
+    description: make target
+    required: true
+
+  attempts:
+    description: Number of runs
+    required: false
+    default: 10
+
+  bin:
+    description: difftest binary location
+    required: false
+    default: ${GITHUB_WORKSPACE}/build.assets/tooling/bin/difftest
+        
+runs:
+  using: "composite"
+  steps:
+    - name: Build
+      shell: bash
+      run: make ${{ inputs.bin }}
+
+    - name: Show
+      id: diffs
+      shell: bash
+      run: |
+        ${{ inputs.bin }} diff --path ${GITHUB_WORKSPACE} --branch remotes/origin/${GITHUB_BASE_REF} ${{ inputs.flags }}
+        echo "subject=$(${{ inputs.bin }} test --path ${GITHUB_WORKSPACE} --branch remotes/origin/${GITHUB_BASE_REF} -u ${{ inputs.flags }})" > $GITHUB_OUTPUT
+
+    - name: Run
+      shell: bash
+      run: >-
+        if [[ -n "${{ steps.diffs.outputs.subject }}" ]] ; then
+          attempt=1 ;
+          while [[ $attempt -le ${{ inputs.attempts }} ]] ; do
```

Comment:
Instead of using the loop here, can we pass a parameter to the `make` target and use standard `-count N` flag to `go test`?